files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -0,0 +1,96 @@
+#! /bin/bash
+
+VERBOSE=no
+
+# Check components
+COMP_LIST="orchagent neighsyncd bgp"
+EXP_STATE="reconciled"
+
+ASSISTANT_SCRIPT="/usr/bin/neighbor_advertiser"
+
+
+function debug()
+{
+    /usr/bin/logger "WARMBOOT_FINALIZER : $1"
```

Comment:
Instead of `WARMBOOT_FINALIZER`, should we just log the script name (i.e., $0)?

Reply:
Because I tweaked the script name to avoid any shell user type warm<tab> and see this script :-)

Reply:
Understood, you want to log the name of the service, not the script. It's just easier to debug a log message when you know the name of the process which logged it.